mlflow/pyfunc/model.py

```diff
@@ -385,14 +397,25 @@ def _save_model_with_class_artifacts_params(
     if streamable is None:
         streamable = python_model.__class__.predict_stream != PythonModel.predict_stream
 
```

Comment:
@annzhang-db Did we add a test case verifying that code/script-based models can be used with streaming prediction?

Reply:
Yes, just added and it passes.